opencog/nlp/chatbot-psi/chatbot.scm

```diff
@@ -18,51 +18,62 @@
     (cancel-all-threads)
     (reset-all-states)
 
-    (let* ((sent-node (car (nlp-parse utterance)))
-           (list-of-words (get-word-list sent-node)))
-
-        (State input-utterance
-            (Reference
-                list-of-words
-                sent-node
-                (Node utterance)
-            )
+    (catch #t
```

Comment:
Mostly the errors come from R2L... right maybe I should remove this catch, at least in the medium/long run we won't need this, having it would hide any real bugs in the code and may also give a wrong impression that the user has entered something wrong